data-science-onramp/data-cleaning/clean_test.py

```diff
@@ -0,0 +1,158 @@
+import os
+import re
+import uuid
+
+from google.cloud import dataproc_v1 as dataproc
+from google.cloud import storage
+import pytest
+
+# Set global variables
+PROJECT = os.environ['GCLOUD_PROJECT']
+REGION = "us-east4"
+CLUSTER_NAME = f'clean-test-{uuid.uuid4()}'
+BUCKET_NAME = f'clean-test-code-{uuid.uuid4()}'
+DESTINATION_BLOB_NAME = "clean.py"
+JOB_FILE_NAME = f'gs://{BUCKET_NAME}/clean.py'
+JOB_DETAILS = {  # Job configuration
+    'placement': {
+        'cluster_name': CLUSTER_NAME
+    },
+    'pyspark_job': {
+        'main_python_file_uri': JOB_FILE_NAME,
+        'args': [
+            PROJECT,
+            BUCKET_NAME,
+            "--test",
+        ],
+        "jar_file_uris": [
+                "gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar"
+        ],
+    },
+}
+CLUSTER_DATA = {  # Create cluster configuration
+    'project_id': PROJECT,
+    'cluster_name': CLUSTER_NAME,
+    'config': {
+        'gce_cluster_config': {
+            'zone_uri': '',
+            "metadata": {
+                "PIP_PACKAGES": "google-cloud-storage"
+            },
+        },
+        'master_config': {
+            'num_instances': 1,
+            'machine_type_uri': 'n1-standard-8'
```

Comment:
Should machine type and num instances also be global variables?

Reply:
@bradmiro

Reply:
I think it's fine to leave it as-is.